src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -84,6 +107,15 @@
  *   &lt;property name=&quot;ignoreOverriddenMethods&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *  public void func1() throws Throwable {} // violation
```

Comment:
This should be indented with 2 spaces to match other examples.

Reply:
Done